VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -62,6 +64,7 @@
                             style="@style/bold_edx_brand_primary_base_text"
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
+                            android:layout_marginTop="3dp"
```

Comment:
I don't think we need the margins now, as the layout width and the number of lines are already fixed, and the area between the header and course title already seems to be sufficient. If there is to a margin though, then it should probably be standardized in `dimens.xml` (as other dimensions in this layout should also be).